wgpu/examples/timestamp-queries/main.rs

```diff
@@ -0,0 +1,324 @@
+//! Sample demonstrating different kinds of timestamp queries.
```

Comment:
This example could likely use some more exposition about what is going on, as someone who has never used them, what information would they need to use them correctly.